### PR TITLE
feat(worker): BulkAttributeRenamer

### PR DESCRIPTION
**Pull request by kirisaki**
# Overview

## What I've done
- [x] AddStringPrefix
- [x] AddStringSufix
- [x] RemoveStringPrefix
- [x] RemoveStringSuffix
- [ ] SelectedAttribute

### CODE REVIEW

**Comment by kirisaki** [comment]
I can't implement `SelectedAttribute` because there are not enough documents to understand how it works.

**Comment by miseyu** [comment]
@kirisaki 
``` json
[
{"hoge": "fuga01", "piyo": "piyo01"},
{"hoge": "fuga02", "piyo": "piyo02"},
]
```
I would like to see a test where everything is properly renamed from hoge to hogehoge in this data.

**Comment by kirisaki** [comment]
Like that test is [here](https://github.com/reearth/reearth-flow/pull/143/files#diff-e05f5158f07a167cb14ae0e93fd5bfb1eeb5cf40db5f2e443d9fe1030cb5a45d).

**Comment by miseyu** [comment]
@kirisaki 
I don't think any of these inputs start with an Array "[]".

worker/tests/fixture/testdata/attribute/renamer/add_prefix/renamer.json
worker/tests/fixture/testdata/attribute/renamer/add_prefix/renamer.json
worker/tests/fixture/testdata/attribute/renamer/add_suffix/renamer.json
worker/tests/fixture/testdata/attribute/renamer/regular_expression_replace/renamer.json
worker/tests/fixture/testdata/attribute/renamer/remove_prefix/renamer.json
worker/tests/fixture/testdata/attribute/renamer/remove_suffix/renamer.json
worker/tests/fixture/testdata/attribute/renamer/string_replace/renamer.json

File: worker/crates/action-universal/src/attribute/renamer.rs [hunk 1]
```diff
@@ -20,30 +23,75 @@ pub(super) enum Rename {
...
     async fn run(&self, _ctx: ActionContext, inputs: Option<ActionDataframe>) -> ActionResult {
         let output = inputs
+            .ok_or(Error::input("no input"))?
+            .into_iter()
+            .map(|(k, v)| {
+                (
+                    if k == *DEFAULT_PORT {
+                        k
+                    } else {
+                        Port::new(rename_key(k.into_inner(), &self.action))
```
Comment (miseyu) [inline worker/crates/action-universal/src/attribute/renamer.rs:53]: Oh, this is different.

Comment (miseyu) [reply worker/crates/action-universal/src/attribute/renamer.rs:53]: @kirisaki 
By attribute, we mean a column of table data.
In other words, this function renames the hashmap key of Array<HashMap>.

**Comment by kirisaki** [comment]
@miseyu I have corrected what you pointed out. Please check it.

**Review by miseyu — APPROVED** [review]
@kirisaki 
対応ありがとうございます！！
LGTM

**Pull request merged**